crates/onnx-ir/src/from_onnx.rs

```diff
@@ -408,40 +411,96 @@ pub(crate) fn remap_unsqueeze_to_reshape(node: &mut Node, out_arg: &Argument) {
         node.node_type = NodeType::Reshape;
     }
 }
-// Define a trait for topological sorting
-trait TopologicalSortable {
-    fn is_top_sorted(&self) -> bool;
-}
 
-impl TopologicalSortable for Vec<NodeProto> {
-    fn is_top_sorted(&self) -> bool {
-        // Create a hashmap to store the position of each node in the vector
-        let position: HashMap<String, usize> = self
-            .iter()
-            .enumerate()
-            .map(|(idx, node)| (node.name.clone(), idx))
-            .collect();
+/// Topographically sort an ONNX node graph.
```

Comment:
Pretty minor, but I think you meant *topologically* and not *topographically*

Reply:
Oops! Let me fix that, thanks :)